zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/event/DocumentInformatieObjectDownloaded.kt

```diff
@@ -0,0 +1,10 @@
+package com.ritense.documentenapi.event
+
+import com.ritense.outbox.domain.BaseEvent
+
+class DocumentInformatieObjectDownloaded (documentInformatieobjectUrl: String) : BaseEvent(
+    type = "com.ritense.gzac.drc.enkelvoudiginformatieobject.downloaded",
+    resultType = "java.io.InputStream",
```

Comment:
Since an `InputStream` can be anything and does not help in regards to understanding the content, maybe it's better to leave it `null`